lib/rouge/lexers/objective_c.rb

```diff
@@ -88,7 +88,6 @@ def self.analyze_text(text)
 
       state :message_shared do
         rule /\]/, Punctuation, :pop!
-        rule /;/, Error
```

Comment:
Hm, would it be possible to push a new state that mixed in `:statements` when it encounters `{` instead? (and pop at `}`)